.github/workflows/ci_workflow.yml

```diff
@@ -225,6 +226,22 @@ jobs:
                 --config ${{ matrix.build-type }} \
                 -- -j4
         working-directory: _build
+      - name: Examples
+        run: |
+          # Build and run the examples as a standalone program linking 
+          # against the OpenEXR isntallation.
+          cmake -B . \
+                -S ../src/examples \
+                -DOpenEXR_DIR=../../_install/lib64/cmake/OpenEXR \
+                -DImath_DIR=../../_install/lib64/cmake/Imath \
```

Comment:
Shouldn't this just be

    -DOpenEXR_ROOT=../../_install \
    -DImath_ROOT=../.../_install \

??

Reply:
I think the better way is -DCMAKE_PREFIX_PATH=../../_install, rather than OpenEXR_DIR/Imath_DIR.